components/pkg-export-kubernetes/defaults/KubernetesManifest.hbs

```diff
@@ -24,6 +24,8 @@ spec:
   ## An object containing parameters that affects how the Habitat service
   ## is executed.
   service:
+    ## Name of the Habitat service.
```

Comment:
hmm maybe Habitat pkg would make more sense to the Habitat users. Maybe leave it like this, and I can bring it up in the upstream PR and we can see what they think makes more sense to them.